src/Modes/Normal.ts

```diff
@@ -50,6 +54,13 @@ export class ModeNormal extends Mode {
                 .then(ActionDelete.byMotions.bind(undefined, {motions: [MotionLine.end()], shouldYank: true}))
                 .then(() => ActionMode.toInsert());
         } },
+        { keys: 'c i {char}', 
+            command: (args: {character: string}) => 
+                ActionMoveCursor.byMotions({motions: [MotionMatchOpening.matchOpening(args, false, true)]})
+                .then(() => ActionDelete.byMotions({motions: [MotionMatchClosing.matchClosing(args, true, false)], shouldYank: true}))
```

Comment:
We can put these `boolean` params into `args` and give them keys, so we don't need to look into the definition to find out what they mean.